### ISSUE TITLE
Confidence Intervals - illegible and most of it not significant

### ISSUE THREAD

**Issue by usr7777** [SachaEpskamp/psychonetrics#35]
Hello Sacha,

I am trying to get the confidence intervals plots:
[CI_beta.pdf](https://github.com/user-attachments/files/16202828/CI_beta.pdf)
[CI_temporal.pdf](https://github.com/user-attachments/files/16202829/CI_temporal.pdf)

The black lines apparently represent a p-value > 0.05. Hence, given these two graphs above, most of it is not significant, right? Besides, the graphs are pretty illegible even when I extra zoom in. Is this normal, please?

**Comment by SachaEpskamp** [comment]
This looks quite usual; there are quite a lot of significant edges. Did you use thresholding when extracting the matrix to plot? e.g., getmatrix(model, "PDC", threshold=TRUE)?

**Comment by usr7777** [comment]
Yes, to extract the temporal network (getmatrix(mod, "PDC", threshold = TRUE)) and the contemporaneous network (getmatrix(mod, "omega_zeta_within", threshold = TRUE)), I used thresholding. Only for the between-person network, I used the saturated model (getmatrix(mod, "omega_zeta_between")). My networks looked like that after thresholding the temporal and contemporaneous networks only (which also seemed a bit strange to me, but I guess maybe that's how it should be):
[thresholded_network.pdf](https://github.com/user-attachments/files/16216496/thresholded_network.pdf)

All the 3 networks, in the saturated form, looked like that, which is worse:
[Networks_trial1.pdf](https://github.com/user-attachments/files/16216499/Networks_trial1.pdf)

For the confidence intervals, I was able to extract only for the temporal and contemporaneous networks, for the betwee-person, I was getting errors.
# Look at CIs:
CIplot(mod, "beta")
CIplot(mod, "omega_zeta_within")
CIplot(mod, "omega_zeta_between")
Error in data.frame(edge = edges, est = est, p = p, lower = ci_lower,  : 
  arguments imply differing number of rows: 1, 0

The model fit seemed okay though:
[Model fit.txt](https://github.com/user-attachments/files/16216527/Model.fit.txt)

**Comment by SachaEpskamp** [comment]
It is pretty common that the between person effects are harder to be estimated. In this case it is best to model these as a Cholesky decomposition and not to interpret the results at the between person level substantially. The rest looks ok here (indeed it is important to threshold as otherwise the networks are hard to interpret).

**Comment by usr7777** [comment]
Thank you very much! :) Best wishes, Shabneez.
